resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/config.scala

```diff
@@ -58,12 +58,43 @@ package object config {
 
   private[spark] val DRIVER_LABELS =
     ConfigBuilder("spark.mesos.driver.labels")
-      .doc("Mesos labels to add to the driver.  Labels are free-form key-value pairs.  Key-value " +
+      .doc("Mesos labels to add to the driver.  Labels are free-form key-value pairs. Key-value " +
         "pairs should be separated by a colon, and commas used to list more than one." +
         "Ex. key:value,key2:value2")
       .stringConf
       .createOptional
 
+  private[spark] val SECRET_NAME =
+    ConfigBuilder("spark.mesos.driver.secret.name")
```

Comment:
Are you going to keep the config names in the singular? That's both odd since they're lists, and inconsistent with the config above ("spark.mesos.driver.labels").

Reply:
I suppose this naming came from the Mesos protos where [secret](https://github.com/apache/mesos/blob/1.3.0/include/mesos/mesos.proto#L2255) and [name](https://github.com/apache/mesos/blob/1.3.0/include/mesos/mesos.proto#L2041) are singular and [volumes](https://github.com/apache/mesos/blob/1.3.0/include/mesos/mesos.proto#L2532) is plural. However the last of which is hidden in this case. I agree `names` is perhaps more intuitive?